From ed30c466c91dc3888352ca11bfc4cb36f34f0265 Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Thu, 25 Jan 2018 13:19:39 +0100
Subject: [PATCH] Fix smartcard usage in manpage

Man page has been updated according to the wiki. Unfortunately, wiki
mentions "smartcard" subplugin, which was introduced later, but
stable-1.0 uses just scard. Let's fix the man page accordingly.
---
 client/X11/xfreerdp.1.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/X11/xfreerdp.1.xml b/client/X11/xfreerdp.1.xml
index 3b1dc675e..039e87d92 100644
--- a/client/X11/xfreerdp.1.xml
+++ b/client/X11/xfreerdp.1.xml
@@ -539,7 +539,7 @@
                       </listitem>
                     </varlistentry>
                     <varlistentry>
-                      <term>smartcard[:<replaceable class="parameter">name</replaceable>]</term>
+                      <term>scard[:<replaceable class="parameter">name</replaceable>]</term>
                       <listitem>
                         <para>
                           Redirect smartcard with <replaceable class="parameter">name</replaceable>.
-- 
2.16.1

